packages/react-router/src/route.tsx

```
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

ESLint

[error] Definition for rule 'react-hooks/rules-of-hooks' was not found.

(react-hooks/rules-of-hooks)